openage/convert/opus/opus.pxd

```diff
@@ -0,0 +1,29 @@
+# Copyright 2016-2017 the openage authors. See copying.md for legal info.
+
+cdef extern from "opus/opus.h":
+    ctypedef struct OpusEncoder:
+        pass
+
+    # TODO replace int by int32_t or hope that this works
+    ctypedef int opus_int32
+    ctypedef short opus_int16
+    OpusEncoder *opus_encoder_create(opus_int32 Fs, int channels,
+                                     int application, int *error)
+    void opus_encoder_destroy(OpusEncoder *st)
+    opus_int32 opus_encode(OpusEncoder *st, const opus_int16 *pcm,
+                           int frame_size, unsigned char *data,
+                           opus_int32 max_data_bytes)
+    int opus_encoder_ctl(OpusEncoder *st, int request)
```

Comment:
opus_encoder takes actually more than two arguments, but one has to use a macro for the second argument, which evaluates to commata and arguments. but there are no macros in cython. what do? see below! just say the macro is a function. cython will just leave the 'function call' there, and that way, the c preprocessor can see it.